app/lib/actions/katello/repository/metadata_generate.rb

```diff
@@ -2,22 +2,16 @@ module Actions
   module Katello
     module Repository
       class MetadataGenerate < Actions::Base
-        include Actions::Katello::PulpSelector
-
         def plan(repository, options = {})
-          dependency = options.fetch(:dependency, nil)
-          force = options.fetch(:force, false)
           source_repository = options.fetch(:source_repository, nil)
           source_repository ||= repository.target_repository if repository.link?
           smart_proxy = options.fetch(:smart_proxy, SmartProxy.pulp_primary)
           matching_content = options.fetch(:matching_content, false)
 
-          plan_pulp_action([Pulp::Repository::DistributorPublish, Pulp3::Orchestration::Repository::GenerateMetadata],
+          plan_action(Pulp3::Orchestration::Repository::GenerateMetadata,
                         repository, smart_proxy,
-                        :force => force,
                         :source_repository => source_repository,
-                        :matching_content => matching_content,
-                        :dependency => dependency)
+                        :matching_content => matching_content)
```

Comment:
i'm kinda questing if we should remove the dependency option, or add it to the Pulp3::Orchestration::Repository::GenerateMetadata task.  I *think* it was used to ensure that if some step A fails, that a later step B would be skipped, but my memory is a bit hazy.  Let me do some testing